godot-core/src/obj/gd.rs

```diff
@@ -416,6 +418,11 @@ impl<T: GodotClass> Gd<T> {
         T::Mem::maybe_init_ref(&self);
         self
     }
+
+    /// Returns a callable referencing a method from this object named `method_name`.
+    pub fn callable<S: Into<StringName>>(&self, method_name: S) -> Callable {
+        Callable::from_object_method(self.share(), method_name)
+    }
```

Comment:
I see the convenience of this method; though over time, as we're adding more and more interactions between `Gd` and other types, we might want to decide where those should live (and I'd tend to "outside of `Gd`").

It's fine for now, but this may still change in the future.

Reply:
Yeah, it's convenient for now but we should have a proper discussion of what the API for using callable should be.